docs/src/main/asciidoc/_developer-guide/Class_Loading_in_WildFly.adoc

```diff
@@ -450,3 +449,103 @@ relevant category to ERROR or higher.
 Other than the WARN messages noted above, declaring a direct dependency
 on a non-public module has no impact on how WildFly processes the
 deployment.
+
+
+[[how-to-list-the-module-dependencies-of-a-deployed-application]]
+== How to list the module dependencies of a deployed application
+
+In Wildfly it is possible to list the module dependencies added by the container to your deployed application. This task can be achieved via the command line interface, where specific commands are available to list the module dependencies for deployments and ear-subdeployments.
```

Comment:
s/commands/operations

In CLI terms a 'command' is a high level command, while an 'operation' is for the stuff that gets parsed into DMR.